Sources/Regex/AST.swift

```diff
@@ -23,21 +24,11 @@ public enum AST: ASTEntity {
   case empty
 }
 
-// MARK: - Convenience constructors
-extension AST {
-  public static func zeroOrMore(
-    _ kind: Quantifier.Kind, _ a: AST
-  ) -> AST {
-    .quantification(.zeroOrMore(kind), a)
-  }
-  public static func oneOrMore(
-    _ kind: Quantifier.Kind, _ a: AST
-  ) -> AST {
-    .quantification(.oneOrMore(kind), a)
-  }
-  public static func zeroOrOne(
-    _ kind: Quantifier.Kind, _ a: AST
-  ) -> AST {
-    .quantification(.zeroOrOne(kind), a)
-  }
+// Note that we're not yet an ASTEntity, would need to be a struct.
```

Comment:
`ASTStorage` is a struct, is this comment outdated?

Reply:
Its outdated, yes. There's a big rift between having natural nested enums vs retained source info and storage. On the other hand, it's possible this rift could be handled at the `ASTAction` level and we don't have storage types, just a different backend. WDYT?